tcms/testcases/static/testcases/js/get.js

```diff
@@ -7,7 +7,7 @@ function addComponent(object_id, _input, to_table) {
     if (_name.length > 0) {
         jsonRPC('TestCase.add_component', [object_id, _name], function(data) {
             if (data !== undefined) {
-                to_table.row.add({name: _name}).draw();
+                to_table.row.add({name: data.name, id: data.id}).draw();
```

Comment:
If I understand this correctly the second argument goes into the second column in the table. What if we want to add more columns or change their order in the future? Can we tie the field name "id" to the column with the delete button somehow (see lines 106-118) ?

Reply:
I went through the documentation of DataTables plugin. According to ["Renderers"](https://www.datatables.net/manual/data/renderers) and [Objects](https://www.datatables.net/manual/data/#Objects) section of their documentation, all we need to do is `data: "id"` instead of `data: null` in definition of the table. I have made this change.